www/include/monitoring/status/Hosts/xml/hostXML.php

```diff
@@ -190,60 +191,63 @@
 }
 
 if ($hostgroups) {
-    $rq1 .= " AND h.host_id = hhg.host_id AND hg.hostgroup_id IN ($hostgroups) AND hhg.hostgroup_id = hg.hostgroup_id";
+    $rq1 .= " AND h.host_id = hhg.host_id AND hg.hostgroup_id IN (" . $hostgroups . ") AND hhg.hostgroup_id = hg
+        .hostgroup_id";
```

Comment:
Why did you put that line underneath it?
Split the line from the $hostgroups variable.

Reply:
thanks, my IDE did it for me ;)